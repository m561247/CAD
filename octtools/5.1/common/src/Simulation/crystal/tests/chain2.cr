model slope
option printedgespeed
build chain2.sim
in In
out Out
del In 0 -1
