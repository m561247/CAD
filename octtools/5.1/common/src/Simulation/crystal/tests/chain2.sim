| units: 1	tech: nmos
e In A GND 400 400 100 100
d A A Vdd 1600 400 200 200
e A B GND 400 400 100 100
d B B Vdd 1600 400 200 200
e Vdd B B2 400 400 700 700
e B2 C GND 400 400 100 100
d C C Vdd 1600 400 200 200
e C D GND 400 400 100 100
d D D Vdd 1600 400 200 200
e D E GND 400 400 100 100
d E E Vdd 1600 400 200 200
e Vdd E E2 400 400 700 700
e E2 Out GND 400 400 100 100
d Out Out Vdd 1600 400 200 200
