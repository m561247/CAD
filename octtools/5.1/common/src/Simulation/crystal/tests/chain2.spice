.WIDTH out=120
* Nodes 4-5 correspond to Out (see gate at 1,1)
* Nodes 6-7 correspond to E2 (see gate at 1,1)
* Nodes 8-9 correspond to E (see gate at 1,1)
* Nodes 10-11 correspond to D (see gate at 1,1)
* Nodes 12-13 correspond to C (see gate at 1,1)
* Nodes 14-15 correspond to B2 (see gate at 1,1)
* Nodes 16-17 correspond to B (see gate at 1,1)
* Nodes 18-19 correspond to A (see gate at 1,1)
* Nodes 20-21 correspond to In (see gate at 1,1)

C1 15 0 0.041pf
C2 19 0 0.990pf
C3 17 0 0.047pf
C4 13 0 0.040pf
C5 11 0 0.190pf
C6 9 0 0.047pf
C7 21 0 0.042pf
C8 7 0 0.041pf
C9 5 0 0.048pf
M1 1 5 5 2 d l=16.0u w=4.0u
M2 0 7 5 2 e l=4.0u w=4.0u
M3 7 1 9 2 e l=4.0u w=4.0u
M4 0 11 9 2 e l=4.0u w=4.0u
M5 1 9 9 2 d l=16.0u w=4.0u
M6 1 11 11 2 d l=16.0u w=4.0u
M7 0 13 11 2 e l=4.0u w=4.0u
M8 0 15 13 2 e l=4.0u w=4.0u
M9 1 13 13 2 d l=16.0u w=4.0u
M10 15 1 17 2 e l=4.0u w=4.0u
M11 1 17 17 2 d l=16.0u w=4.0u
M12 0 19 17 2 e l=4.0u w=4.0u
M13 0 21 19 2 e l=4.0u w=4.0u
M14 1 19 19 2 d l=16.0u w=4.0u

* Initial conditions:
.ic v(15)=0.000000
.ic v(19)=5.000000
.ic v(17)=0.000000
.ic v(13)=5.000000
.ic v(11)=0.000000
.ic v(9)=5.000000
.ic v(21)=0.000000
.ic v(7)=5.000000
.ic v(5)=0.000000

vin 21 0 pulse(0 5 0ns 0ns 0ns)
.tran 0.4ns 80ns
.print tran V(5) V(7) V(11) V(13) V(15) V(19) (0,5)
.end
