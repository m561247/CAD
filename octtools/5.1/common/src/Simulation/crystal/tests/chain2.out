1*******08/21/83 ********  SPICE 2G.6    3/15/83 ********00:00:54*****

0SPICE DECK GENERATED BY CRYSTAL.                                                

0****     INPUT LISTING                    TEMPERATURE =   27.000 DEG C

0***********************************************************************




 ** THE FOLLOWING LINES DEFINE THE NMOS MODELS AND VDD.  

 .MODEL E NMOS(VTO=.9 NSUB=2E15 TOX=.1U UO=800 XQC=.5    
  +LEVEL=2 VMAX=1E5 CGSO=245P CGDO=245P) 
 .MODEL D NMOS(VTO=-3.2 NSUB=2E15 TOX=.1U UO=800 XQC=.5  
  +LEVEL=2 VMAX=1E5 CGSO=245P CGDO=245P) 
 .OPTIONS NOMOD  
 .WIDTH OUT=80   
 VDD 1 0 5       
 VBB 2 0 0       
 .WIDTH OUT=120  
 * NODES 4-5 CORRESPOND TO OUT (SEE GATE AT 1,1) 
 * NODES 6-7 CORRESPOND TO E2 (SEE GATE AT 1,1)  
 * NODES 8-9 CORRESPOND TO E (SEE GATE AT 1,1)   
 * NODES 10-11 CORRESPOND TO D (SEE GATE AT 1,1) 
 * NODES 12-13 CORRESPOND TO C (SEE GATE AT 1,1) 
 * NODES 14-15 CORRESPOND TO B2 (SEE GATE AT 1,1)
 * NODES 16-17 CORRESPOND TO B (SEE GATE AT 1,1) 
 * NODES 18-19 CORRESPOND TO A (SEE GATE AT 1,1) 
 * NODES 20-21 CORRESPOND TO IN (SEE GATE AT 1,1)
         
 C1 15 0 0.041PF 
 C2 19 0 0.990PF 
 C3 17 0 0.047PF 
 C4 13 0 0.040PF 
 C5 11 0 0.190PF 
 C6 9 0 0.047PF  
 C7 21 0 0.042PF 
 C8 7 0 0.041PF  
 C9 5 0 0.048PF  
 M1 1 5 5 2 D L=16.0U W=4.0U     
 M2 0 7 5 2 E L=4.0U W=4.0U      
 M3 7 1 9 2 E L=4.0U W=4.0U      
 M4 0 11 9 2 E L=4.0U W=4.0U     
 M5 1 9 9 2 D L=16.0U W=4.0U     
 M6 1 11 11 2 D L=16.0U W=4.0U   
 M7 0 13 11 2 E L=4.0U W=4.0U    
 M8 0 15 13 2 E L=4.0U W=4.0U    
 M9 1 13 13 2 D L=16.0U W=4.0U   
 M10 15 1 17 2 E L=4.0U W=4.0U   
 M11 1 17 17 2 D L=16.0U W=4.0U  
 M12 0 19 17 2 E L=4.0U W=4.0U   
 M13 0 21 19 2 E L=4.0U W=4.0U   
 M14 1 19 19 2 D L=16.0U W=4.0U  
         
 * INITIAL CONDITIONS:   
 .IC V(15)=0.000000      
 .IC V(19)=5.000000      
 .IC V(17)=0.000000      
 .IC V(13)=5.000000      
 .IC V(11)=0.000000      
 .IC V(9)=5.000000       
 .IC V(21)=0.000000      
 .IC V(7)=5.000000       
 .IC V(5)=0.000000       
         
 VIN 21 0 PULSE(0 5 0NS 0NS 0NS) 
 .TRAN 0.4NS 80NS
 .PRINT TRAN V(5) V(7) V(11) V(13) V(15) V(19) (0,5)     
 .END    
1****************08/21/83 ************************  SPICE 2G.6    3/15/83 ************************00:00:54****************

0SPICE DECK GENERATED BY CRYSTAL.                                                
0****                 INITIAL TRANSIENT SOLUTION                           TEMPERATURE =   27.000 DEG C

0*************************************************************************************************************************


  NODE   VOLTAGE     NODE   VOLTAGE     NODE   VOLTAGE     NODE   VOLTAGE     NODE   VOLTAGE     NODE   VOLTAGE


 (  1)    5.0000    (  2)    0.        (  5)    0.0000    (  7)    5.0000    (  9)    5.0000    ( 11)    0.0000

 ( 13)    5.0000    ( 15)    0.0000    ( 17)    0.0000    ( 19)    5.0000    ( 21)    0.        




     VOLTAGE SOURCE CURRENTS

     NAME       CURRENT


     VDD      -8.443d-05

     VBB       9.706d-11

     VIN       0.   d+00


     TOTAL POWER DISSIPATION   4.22d-04  WATTS
1****************08/21/83 ************************  SPICE 2G.6    3/15/83 ************************00:00:54****************

0SPICE DECK GENERATED BY CRYSTAL.                                                
0****                 OPERATING POINT INFORMATION                          TEMPERATURE =   27.000 DEG C

0*************************************************************************************************************************


0
0**** MOSFETS


0            M1        M2        M3        M4        M5        M6        M7        M8        M9        M10     
0MODEL     D         E         E         E         D         D         E         E         D         E       
 ID        2.81d-05 -3.19d-09  1.93d-12  0.  d+00  1.93d-12  2.81d-05 -3.19d-09  0.  d+00  1.93d-12 -3.18d-09
 VGS          0.        5.000     0.000    -5.000     0.        0.        5.000    -5.000     0.        5.000
 VDS          5.000    -0.000     0.000    -5.000     0.000     5.000    -0.000    -5.000     0.000    -0.000
 VBS         -0.000    -0.000    -5.000    -5.000    -5.000    -0.000    -0.000    -5.000    -5.000    -0.000


0            M11       M12       M13       M14     
0MODEL     D         E         E         D       
 ID        2.81d-05 -3.19d-09  0.  d+00  1.93d-12
 VGS          0.        5.000    -5.000     0.   
 VDS          5.000    -0.000    -5.000     0.000
 VBS         -0.000    -0.000    -5.000    -5.000
1****************08/21/83 ************************  SPICE 2G.6    3/15/83 ************************00:00:54****************

0SPICE DECK GENERATED BY CRYSTAL.                                                
0****                 TRANSIENT ANALYSIS                                   TEMPERATURE =   27.000 DEG C

0*************************************************************************************************************************



     TIME       V(5)        V(7)        V(11)       V(13)       V(15)       V(19)       
X
 
  0.   d+00     2.814d-05   5.000d+00   2.814d-05   5.000d+00   3.187d-09   5.000d+00
  4.000d-10     1.425d-01   5.011d+00   4.924d-02   5.003d+00   4.073d-02   4.980d+00
  8.000d-10     2.002d-01   5.015d+00   8.894d-02   5.006d+00   9.478d-02   4.896d+00
  1.200d-09     2.255d-01   5.017d+00   1.198d-01   5.008d+00   1.343d-01   4.812d+00
  1.600d-09     2.382d-01   5.018d+00   1.448d-01   5.009d+00   1.656d-01   4.730d+00
  2.000d-09     2.426d-01   5.018d+00   1.644d-01   5.010d+00   1.887d-01   4.648d+00
  2.400d-09     2.457d-01   5.018d+00   1.816d-01   5.010d+00   2.092d-01   4.567d+00
  2.800d-09     2.460d-01   5.018d+00   1.934d-01   5.010d+00   2.236d-01   4.488d+00
  3.200d-09     2.463d-01   5.018d+00   2.052d-01   5.010d+00   2.380d-01   4.408d+00
  3.600d-09     2.463d-01   5.018d+00   2.122d-01   5.010d+00   2.484d-01   4.331d+00
  4.000d-09     2.463d-01   5.018d+00   2.182d-01   5.009d+00   2.580d-01   4.254d+00
  4.400d-09     2.463d-01   5.018d+00   2.242d-01   5.009d+00   2.676d-01   4.178d+00
  4.800d-09     2.463d-01   5.018d+00   2.302d-01   5.009d+00   2.772d-01   4.101d+00
  5.200d-09     2.463d-01   5.018d+00   2.332d-01   5.008d+00   2.855d-01   4.026d+00
  5.600d-09     2.463d-01   5.018d+00   2.356d-01   5.008d+00   2.935d-01   3.953d+00
  6.000d-09     2.463d-01   5.018d+00   2.379d-01   5.008d+00   3.015d-01   3.879d+00
  6.400d-09     2.463d-01   5.018d+00   2.403d-01   5.007d+00   3.095d-01   3.805d+00
  6.800d-09     2.463d-01   5.018d+00   2.415d-01   5.007d+00   3.179d-01   3.733d+00
  7.200d-09     2.463d-01   5.018d+00   2.425d-01   5.006d+00   3.264d-01   3.662d+00
  7.600d-09     2.463d-01   5.018d+00   2.434d-01   5.006d+00   3.349d-01   3.591d+00
  8.000d-09     2.463d-01   5.018d+00   2.444d-01   5.006d+00   3.434d-01   3.520d+00
  8.400d-09     2.463d-01   5.018d+00   2.449d-01   5.005d+00   3.529d-01   3.452d+00
  8.800d-09     2.463d-01   5.018d+00   2.453d-01   5.005d+00   3.626d-01   3.383d+00
  9.200d-09     2.463d-01   5.018d+00   2.456d-01   5.005d+00   3.724d-01   3.315d+00
  9.600d-09     2.463d-01   5.018d+00   2.460d-01   5.005d+00   3.821d-01   3.246d+00
  1.000d-08     2.463d-01   5.018d+00   2.463d-01   5.004d+00   3.936d-01   3.180d+00
  1.040d-08     2.463d-01   5.018d+00   2.465d-01   5.004d+00   4.055d-01   3.114d+00
  1.080d-08     2.463d-01   5.018d+00   2.466d-01   5.004d+00   4.173d-01   3.048d+00
  1.120d-08     2.463d-01   5.018d+00   2.468d-01   5.004d+00   4.292d-01   2.982d+00
  1.160d-08     2.463d-01   5.018d+00   2.469d-01   5.004d+00   4.433d-01   2.917d+00
  1.200d-08     2.463d-01   5.018d+00   2.470d-01   5.004d+00   4.579d-01   2.854d+00
  1.240d-08     2.463d-01   5.018d+00   2.470d-01   5.004d+00   4.726d-01   2.790d+00
  1.280d-08     2.463d-01   5.018d+00   2.471d-01   5.003d+00   4.872d-01   2.727d+00
  1.320d-08     2.463d-01   5.018d+00   2.471d-01   5.003d+00   5.052d-01   2.665d+00
  1.360d-08     2.463d-01   5.018d+00   2.471d-01   5.003d+00   5.239d-01   2.604d+00
  1.400d-08     2.463d-01   5.018d+00   2.471d-01   5.003d+00   5.426d-01   2.543d+00
  1.440d-08     2.463d-01   5.018d+00   2.471d-01   5.003d+00   5.613d-01   2.482d+00
  1.480d-08     2.463d-01   5.018d+00   2.471d-01   5.003d+00   5.852d-01   2.423d+00
  1.520d-08     2.463d-01   5.018d+00   2.471d-01   5.004d+00   6.100d-01   2.365d+00
  1.560d-08     2.463d-01   5.018d+00   2.471d-01   5.004d+00   6.348d-01   2.306d+00
  1.600d-08     2.463d-01   5.018d+00   2.470d-01   5.004d+00   6.597d-01   2.248d+00
  1.640d-08     2.463d-01   5.018d+00   2.470d-01   5.004d+00   6.926d-01   2.192d+00
  1.680d-08     2.463d-01   5.018d+00   2.470d-01   5.004d+00   7.270d-01   2.137d+00
  1.720d-08     2.463d-01   5.018d+00   2.470d-01   5.004d+00   7.615d-01   2.082d+00
  1.760d-08     2.463d-01   5.018d+00   2.469d-01   5.004d+00   7.959d-01   2.027d+00
  1.800d-08     2.463d-01   5.018d+00   2.469d-01   5.004d+00   8.402d-01   1.975d+00
  1.840d-08     2.463d-01   5.018d+00   2.470d-01   5.004d+00   8.863d-01   1.924d+00
  1.880d-08     2.463d-01   5.018d+00   2.470d-01   5.004d+00   9.324d-01   1.872d+00
  1.920d-08     2.463d-01   5.018d+00   2.471d-01   5.004d+00   9.786d-01   1.821d+00
  1.960d-08     2.463d-01   5.018d+00   2.472d-01   5.001d+00   1.033d+00   1.773d+00
  2.000d-08     2.463d-01   5.018d+00   2.472d-01   4.997d+00   1.089d+00   1.725d+00
  2.040d-08     2.463d-01   5.018d+00   2.473d-01   4.993d+00   1.144d+00   1.678d+00
  2.080d-08     2.463d-01   5.018d+00   2.474d-01   4.989d+00   1.200d+00   1.631d+00
  2.120d-08     2.463d-01   5.018d+00   2.476d-01   4.975d+00   1.261d+00   1.587d+00
  2.160d-08     2.463d-01   5.018d+00   2.478d-01   4.959d+00   1.323d+00   1.543d+00
  2.200d-08     2.463d-01   5.018d+00   2.480d-01   4.943d+00   1.384d+00   1.500d+00
  2.240d-08     2.463d-01   5.018d+00   2.482d-01   4.927d+00   1.446d+00   1.457d+00
  2.280d-08     2.463d-01   5.018d+00   2.489d-01   4.895d+00   1.510d+00   1.417d+00
  2.320d-08     2.463d-01   5.018d+00   2.497d-01   4.859d+00   1.575d+00   1.378d+00
  2.360d-08     2.463d-01   5.018d+00   2.504d-01   4.823d+00   1.639d+00   1.339d+00
  2.400d-08     2.463d-01   5.018d+00   2.512d-01   4.787d+00   1.704d+00   1.300d+00
  2.440d-08     2.463d-01   5.018d+00   2.530d-01   4.730d+00   1.768d+00   1.264d+00
  2.480d-08     2.463d-01   5.018d+00   2.550d-01   4.669d+00   1.833d+00   1.229d+00
  2.520d-08     2.463d-01   5.018d+00   2.570d-01   4.608d+00   1.897d+00   1.194d+00
  2.560d-08     2.463d-01   5.018d+00   2.590d-01   4.547d+00   1.962d+00   1.159d+00
  2.600d-08     2.463d-01   5.018d+00   2.624d-01   4.462d+00   2.024d+00   1.128d+00
  2.640d-08     2.463d-01   5.018d+00   2.661d-01   4.372d+00   2.086d+00   1.097d+00
  2.680d-08     2.463d-01   5.018d+00   2.698d-01   4.282d+00   2.148d+00   1.066d+00
  2.720d-08     2.463d-01   5.018d+00   2.735d-01   4.192d+00   2.210d+00   1.035d+00
  2.760d-08     2.463d-01   5.018d+00   2.793d-01   4.076d+00   2.267d+00   1.007d+00
  2.800d-08     2.463d-01   5.018d+00   2.856d-01   3.956d+00   2.323d+00   9.792d-01
  2.840d-08     2.463d-01   5.018d+00   2.919d-01   3.836d+00   2.380d+00   9.518d-01
  2.880d-08     2.463d-01   5.018d+00   2.982d-01   3.715d+00   2.436d+00   9.244d-01
  2.920d-08     2.463d-01   5.018d+00   3.080d-01   3.570d+00   2.484d+00   8.999d-01
  2.960d-08     2.463d-01   5.018d+00   3.184d-01   3.419d+00   2.531d+00   8.759d-01
  3.000d-08     2.463d-01   5.018d+00   3.288d-01   3.268d+00   2.578d+00   8.519d-01
  3.040d-08     2.463d-01   5.018d+00   3.392d-01   3.117d+00   2.624d+00   8.279d-01
  3.080d-08     2.463d-01   5.018d+00   3.554d-01   2.950d+00   2.659d+00   8.065d-01
  3.120d-08     2.463d-01   5.018d+00   3.726d-01   2.779d+00   2.692d+00   7.856d-01
  3.160d-08     2.463d-01   5.018d+00   3.898d-01   2.609d+00   2.725d+00   7.647d-01
  3.200d-08     2.463d-01   5.018d+00   4.071d-01   2.438d+00   2.757d+00   7.438d-01
  3.240d-08     2.463d-01   5.018d+00   4.342d-01   2.269d+00   2.780d+00   7.252d-01
  3.280d-08     2.463d-01   5.018d+00   4.633d-01   2.100d+00   2.801d+00   7.071d-01
  3.320d-08     2.463d-01   5.018d+00   4.924d-01   1.931d+00   2.822d+00   6.890d-01
  3.360d-08     2.463d-01   5.018d+00   5.215d-01   1.762d+00   2.843d+00   6.709d-01
  3.400d-08     2.463d-01   5.018d+00   5.617d-01   1.613d+00   2.858d+00   6.548d-01
  3.440d-08     2.463d-01   5.018d+00   6.041d-01   1.467d+00   2.871d+00   6.392d-01
  3.480d-08     2.463d-01   5.018d+00   6.465d-01   1.321d+00   2.885d+00   6.235d-01
  3.520d-08     2.463d-01   5.018d+00   6.890d-01   1.175d+00   2.898d+00   6.079d-01
  3.560d-08     2.463d-01   5.018d+00   7.360d-01   1.072d+00   2.908d+00   5.941d-01
  3.600d-08     2.463d-01   5.018d+00   7.840d-01   9.780d-01   2.918d+00   5.806d-01
  3.640d-08     2.463d-01   5.018d+00   8.319d-01   8.837d-01   2.927d+00   5.671d-01
  3.680d-08     2.463d-01   5.018d+00   8.799d-01   7.894d-01   2.936d+00   5.537d-01
  3.720d-08     2.463d-01   5.018d+00   9.272d-01   7.400d-01   2.944d+00   5.418d-01
  3.760d-08     2.463d-01   5.018d+00   9.743d-01   6.994d-01   2.953d+00   5.303d-01
  3.800d-08     2.463d-01   5.018d+00   1.021d+00   6.587d-01   2.961d+00   5.188d-01
  3.840d-08     2.463d-01   5.018d+00   1.069d+00   6.181d-01   2.969d+00   5.072d-01
  3.880d-08     2.463d-01   5.018d+00   1.115d+00   6.003d-01   2.977d+00   4.971d-01
  3.920d-08     2.463d-01   5.018d+00   1.160d+00   5.869d-01   2.984d+00   4.872d-01
  3.960d-08     2.463d-01   5.018d+00   1.206d+00   5.735d-01   2.992d+00   4.774d-01
  4.000d-08     2.463d-01   5.018d+00   1.252d+00   5.601d-01   2.999d+00   4.675d-01
  4.040d-08     2.463d-01   5.018d+00   1.296d+00   5.537d-01   3.005d+00   4.589d-01
  4.080d-08     2.463d-01   5.018d+00   1.340d+00   5.488d-01   3.012d+00   4.505d-01
  4.120d-08     2.463d-01   5.018d+00   1.385d+00   5.438d-01   3.018d+00   4.421d-01
  4.160d-08     2.463d-01   5.018d+00   1.429d+00   5.389d-01   3.024d+00   4.337d-01
  4.200d-08     2.463d-01   5.018d+00   1.472d+00   5.359d-01   3.029d+00   4.263d-01
  4.240d-08     2.463d-01   5.018d+00   1.515d+00   5.333d-01   3.035d+00   4.192d-01
  4.280d-08     2.463d-01   5.018d+00   1.558d+00   5.307d-01   3.040d+00   4.120d-01
  4.320d-08     2.463d-01   5.018d+00   1.601d+00   5.281d-01   3.045d+00   4.049d-01
  4.360d-08     2.463d-01   5.018d+00   1.643d+00   5.262d-01   3.049d+00   3.986d-01
  4.400d-08     2.463d-01   5.018d+00   1.684d+00   5.244d-01   3.053d+00   3.925d-01
  4.440d-08     2.463d-01   5.018d+00   1.726d+00   5.226d-01   3.057d+00   3.865d-01
  4.480d-08     2.463d-01   5.018d+00   1.767d+00   5.209d-01   3.062d+00   3.804d-01
  4.520d-08     2.463d-01   5.018d+00   1.808d+00   5.194d-01   3.065d+00   3.751d-01
  4.560d-08     2.463d-01   5.018d+00   1.848d+00   5.181d-01   3.069d+00   3.699d-01
  4.600d-08     2.463d-01   5.018d+00   1.888d+00   5.167d-01   3.072d+00   3.648d-01
  4.640d-08     2.463d-01   5.018d+00   1.929d+00   5.153d-01   3.076d+00   3.596d-01
  4.680d-08     2.463d-01   5.018d+00   1.968d+00   5.142d-01   3.079d+00   3.551d-01
  4.720d-08     2.463d-01   5.018d+00   2.007d+00   5.131d-01   3.082d+00   3.508d-01
  4.760d-08     2.463d-01   5.018d+00   2.046d+00   5.120d-01   3.084d+00   3.464d-01
  4.800d-08     2.463d-01   5.018d+00   2.086d+00   5.109d-01   3.087d+00   3.421d-01
  4.840d-08     2.463d-01   5.018d+00   2.124d+00   5.099d-01   3.090d+00   3.383d-01
  4.880d-08     2.463d-01   5.018d+00   2.162d+00   5.090d-01   3.092d+00   3.346d-01
  4.920d-08     2.463d-01   5.018d+00   2.200d+00   5.081d-01   3.095d+00   3.309d-01
  4.960d-08     2.463d-01   5.018d+00   2.238d+00   5.072d-01   3.097d+00   3.272d-01
  5.000d-08     2.463d-01   5.018d+00   2.275d+00   5.064d-01   3.100d+00   3.240d-01
  5.040d-08     2.463d-01   5.018d+00   2.312d+00   5.057d-01   3.102d+00   3.209d-01
  5.080d-08     2.463d-01   5.018d+00   2.349d+00   5.049d-01   3.104d+00   3.178d-01
  5.120d-08     2.463d-01   5.018d+00   2.386d+00   5.041d-01   3.106d+00   3.146d-01
  5.160d-08     2.463d-01   5.015d+00   2.422d+00   5.035d-01   3.108d+00   3.119d-01
  5.200d-08     2.464d-01   5.012d+00   2.458d+00   5.028d-01   3.110d+00   3.093d-01
  5.240d-08     2.464d-01   5.008d+00   2.494d+00   5.021d-01   3.112d+00   3.067d-01
  5.280d-08     2.465d-01   5.005d+00   2.530d+00   5.015d-01   3.114d+00   3.040d-01
  5.320d-08     2.470d-01   4.976d+00   2.565d+00   5.009d-01   3.115d+00   3.018d-01
  5.360d-08     2.477d-01   4.942d+00   2.600d+00   5.003d-01   3.117d+00   2.995d-01
  5.400d-08     2.483d-01   4.908d+00   2.636d+00   4.996d-01   3.119d+00   2.973d-01
  5.440d-08     2.490d-01   4.874d+00   2.671d+00   4.990d-01   3.121d+00   2.951d-01
  5.480d-08     2.521d-01   4.790d+00   2.705d+00   4.986d-01   3.122d+00   2.932d-01
  5.520d-08     2.558d-01   4.696d+00   2.740d+00   4.981d-01   3.124d+00   2.913d-01
  5.560d-08     2.594d-01   4.601d+00   2.774d+00   4.976d-01   3.125d+00   2.894d-01
  5.600d-08     2.631d-01   4.507d+00   2.808d+00   4.971d-01   3.126d+00   2.875d-01
  5.640d-08     2.719d-01   4.365d+00   2.842d+00   4.967d-01   3.128d+00   2.859d-01
  5.680d-08     2.817d-01   4.214d+00   2.876d+00   4.963d-01   3.129d+00   2.843d-01
  5.720d-08     2.915d-01   4.063d+00   2.909d+00   4.958d-01   3.130d+00   2.828d-01
  5.760d-08     3.013d-01   3.911d+00   2.943d+00   4.954d-01   3.132d+00   2.812d-01
  5.800d-08     3.202d-01   3.726d+00   2.976d+00   4.950d-01   3.133d+00   2.798d-01
  5.840d-08     3.409d-01   3.534d+00   3.009d+00   4.947d-01   3.134d+00   2.785d-01
  5.880d-08     3.615d-01   3.342d+00   3.042d+00   4.943d-01   3.135d+00   2.772d-01
  5.920d-08     3.822d-01   3.150d+00   3.075d+00   4.939d-01   3.136d+00   2.758d-01
  5.960d-08     4.228d-01   2.953d+00   3.107d+00   4.935d-01   3.138d+00   2.747d-01
  6.000d-08     4.673d-01   2.755d+00   3.139d+00   4.932d-01   3.139d+00   2.736d-01
  6.040d-08     5.119d-01   2.558d+00   3.171d+00   4.928d-01   3.140d+00   2.724d-01
  6.080d-08     5.564d-01   2.360d+00   3.203d+00   4.925d-01   3.141d+00   2.713d-01
  6.120d-08     6.464d-01   2.189d+00   3.233d+00   4.921d-01   3.142d+00   2.704d-01
  6.160d-08     7.453d-01   2.022d+00   3.264d+00   4.918d-01   3.143d+00   2.694d-01
  6.200d-08     8.441d-01   1.856d+00   3.294d+00   4.914d-01   3.144d+00   2.685d-01
  6.240d-08     9.430d-01   1.689d+00   3.325d+00   4.911d-01   3.145d+00   2.675d-01
  6.280d-08     1.084d+00   1.554d+00   3.354d+00   4.908d-01   3.146d+00   2.667d-01
  6.320d-08     1.233d+00   1.424d+00   3.383d+00   4.905d-01   3.147d+00   2.659d-01
  6.360d-08     1.383d+00   1.295d+00   3.412d+00   4.902d-01   3.148d+00   2.651d-01
  6.400d-08     1.532d+00   1.166d+00   3.440d+00   4.900d-01   3.148d+00   2.643d-01
  6.440d-08     1.687d+00   1.069d+00   3.468d+00   4.897d-01   3.149d+00   2.636d-01
  6.480d-08     1.843d+00   9.791d-01   3.496d+00   4.894d-01   3.150d+00   2.629d-01
  6.520d-08     1.999d+00   8.891d-01   3.523d+00   4.892d-01   3.151d+00   2.623d-01
  6.560d-08     2.155d+00   7.991d-01   3.551d+00   4.889d-01   3.152d+00   2.616d-01
  6.600d-08     2.300d+00   7.405d-01   3.578d+00   4.887d-01   3.152d+00   2.610d-01
  6.640d-08     2.444d+00   6.879d-01   3.605d+00   4.884d-01   3.153d+00   2.605d-01
  6.680d-08     2.587d+00   6.354d-01   3.631d+00   4.882d-01   3.154d+00   2.599d-01
  6.720d-08     2.731d+00   5.828d-01   3.658d+00   4.880d-01   3.155d+00   2.594d-01
  6.760d-08     2.862d+00   5.511d-01   3.684d+00   4.877d-01   3.155d+00   2.589d-01
  6.800d-08     2.991d+00   5.234d-01   3.710d+00   4.875d-01   3.156d+00   2.584d-01
  6.840d-08     3.120d+00   4.957d-01   3.735d+00   4.873d-01   3.157d+00   2.580d-01
  6.880d-08     3.249d+00   4.680d-01   3.761d+00   4.870d-01   3.157d+00   2.575d-01
  6.920d-08     3.362d+00   4.514d-01   3.786d+00   4.868d-01   3.158d+00   2.571d-01
  6.960d-08     3.473d+00   4.371d-01   3.810d+00   4.866d-01   3.159d+00   2.567d-01
  7.000d-08     3.584d+00   4.227d-01   3.834d+00   4.863d-01   3.159d+00   2.563d-01
  7.040d-08     3.694d+00   4.084d-01   3.859d+00   4.861d-01   3.160d+00   2.558d-01
  7.080d-08     3.787d+00   3.993d-01   3.882d+00   4.859d-01   3.161d+00   2.555d-01
  7.120d-08     3.876d+00   3.912d-01   3.905d+00   4.856d-01   3.162d+00   2.552d-01
  7.160d-08     3.965d+00   3.831d-01   3.928d+00   4.854d-01   3.162d+00   2.548d-01
  7.200d-08     4.054d+00   3.750d-01   3.951d+00   4.852d-01   3.163d+00   2.545d-01
  7.240d-08     4.125d+00   3.694d-01   3.973d+00   4.850d-01   3.164d+00   2.542d-01
  7.280d-08     4.193d+00   3.643d-01   3.995d+00   4.849d-01   3.164d+00   2.540d-01
  7.320d-08     4.262d+00   3.592d-01   4.017d+00   4.848d-01   3.165d+00   2.537d-01
  7.360d-08     4.330d+00   3.540d-01   4.038d+00   4.846d-01   3.165d+00   2.534d-01
  7.400d-08     4.383d+00   3.501d-01   4.059d+00   4.845d-01   3.166d+00   2.532d-01
  7.440d-08     4.433d+00   3.464d-01   4.079d+00   4.843d-01   3.166d+00   2.529d-01
  7.480d-08     4.484d+00   3.427d-01   4.100d+00   4.842d-01   3.167d+00   2.527d-01
  7.520d-08     4.534d+00   3.390d-01   4.120d+00   4.840d-01   3.167d+00   2.525d-01
  7.560d-08     4.572d+00   3.359d-01   4.139d+00   4.839d-01   3.167d+00   2.523d-01
  7.600d-08     4.608d+00   3.330d-01   4.158d+00   4.837d-01   3.168d+00   2.520d-01
  7.640d-08     4.644d+00   3.300d-01   4.177d+00   4.836d-01   3.168d+00   2.518d-01
  7.680d-08     4.680d+00   3.271d-01   4.196d+00   4.834d-01   3.169d+00   2.516d-01
  7.720d-08     4.707d+00   3.246d-01   4.214d+00   4.833d-01   3.169d+00   2.515d-01
  7.760d-08     4.733d+00   3.222d-01   4.231d+00   4.831d-01   3.170d+00   2.513d-01
  7.800d-08     4.758d+00   3.197d-01   4.249d+00   4.830d-01   3.170d+00   2.511d-01
  7.840d-08     4.783d+00   3.173d-01   4.266d+00   4.829d-01   3.171d+00   2.510d-01
  7.880d-08     4.802d+00   3.152d-01   4.283d+00   4.827d-01   3.171d+00   2.508d-01
  7.920d-08     4.819d+00   3.132d-01   4.299d+00   4.826d-01   3.171d+00   2.507d-01
  7.960d-08     4.837d+00   3.111d-01   4.316d+00   4.824d-01   3.172d+00   2.506d-01
  8.000d-08     4.854d+00   3.090d-01   4.332d+00   4.823d-01   3.172d+00   2.504d-01
Y
0
         JOB CONCLUDED
0         TOTAL JOB TIME           56.90
